app/models/workers_scaling/algorithms/resources_usage_minimization.rb

```diff
@@ -21,33 +21,32 @@ def self.description
     TOLERANCE = 10
```

Comment:
Zbyt ogolna nazwa. Tolerancja powinna czegos dotyczyc. W komentarzu jest to opisane ale przy uzyciu tego pola w kodzie nie widac komentarza.